hadoop-hdds/docs/content/feature/Decommission.md

```diff
@@ -77,15 +77,16 @@ Storage Container Manager (SCM) decommissioning is the process in which you can
 
 To decommission a SCM and remove the node from the SCM HA ring, the following steps need to be executed.
 ```shell
-ozone admin scm decommission [-hV] [-id=<scmServiceId>] -nodeid=<nodeId>
-                                    [--scm=<scm>]
+ozone admin scm decommission [-hV] [-id=<scmServiceId>] [--scm=<scm>] -nodeid=<nodeId>
 ```
 You can obtain the 'nodeId' by executing this command, **"ozone admin scm roles"**
 
-**Note -** If you want to decommission a **primordial** scm, you may first transfer the leadership to a non primordial scm and then decommission the primordial scm.
+**Note -** If you want to decommission the **leader** scm, you must first transfer the leadership to a different scm and then decommission the node.
 
 To transfer the leader, we can excute below command,
 ```shell
 ozone admin scm transfer --service-id=scmservice -n ${nodeId}
```

Comment:
Should we write _scmservice_ also in some placeholder as this is not fixed right?

Reply:
Done!